### PR TITLE
fix: added testnet badge in app nav while signing

### LINKED ISSUE #3034: Show testnet badge in app nav while signing transaction
It doesn't appear while signing:

image

Though it does while on the home screen:

image

### PR DESCRIPTION
**Pull request by mihirKachroo**
## Description

Added network mode badge in app nav while signing transactions so user can know when they are in mainnet or testnet mode.

To fit badge while maintaining responsiveness of extension, had to decrease fontsize of address and balance text.

Fixes #3034 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by kyranjamie** [comment]
Thanks for the contributions @mihirKachroo—we're busy with some other feature work right now, but aim to get these reviewed/merged as soon as possible.

File: src/app/components/balance.tsx [hunk 1]
```diff
@@ -20,5 +20,5 @@ export function Balance(props: BalanceProps) {
     [balances]
   );
 
-  return <Caption>{balance}</Caption>;
+  return <Caption variant="c3">{balance}</Caption>;
```
Comment (fbwoolf) [inline src/app/components/balance.tsx]: Why this change here?

**Comment by fbwoolf** [comment]
Can you provide a screenshot of the change made?

**Comment by fbwoolf** [comment]
It appears you need to run `yarn:lint:prettier:fix`?

File: src/app/components/balance.tsx (hunk 1 above)
Comment (mihirKachroo) [reply src/app/components/balance.tsx]: To fit the testnet badge into the app nav in a more visually appealing, I wanted to make that balance text a little smaller.

Original:
image

Updated:
image

**Comment by mihirKachroo** [comment]
UI Change:

image

File: src/app/components/balance.tsx (hunk 1 above)
Comment (fbwoolf) [reply src/app/components/balance.tsx]: @mica000 is this ok? @markmhx? Seems small to me.

Comment (markmhendrickson) [reply src/app/components/balance.tsx]: I don't have much of a preference either way. @mica000 wdyt?

**Comment by markmhendrickson** [comment]
@fbwoolf ready to merge?

**Comment by kyranjamie** [comment]
Text should align to the baseline

image

File: src/app/components/balance.tsx (hunk 1 above)
Comment (mihirKachroo) [reply src/app/components/balance.tsx]: I can also make it a little bigger to match the STX address:

image

**Comment by mihirKachroo** [comment]
Is this good?

image

File: src/app/components/balance.tsx (hunk 1 above)
Comment (mica000) [reply src/app/components/balance.tsx]: What is the size of font you are using to display the value @mihirKachroo?

**Comment by mica000** [comment]
> Is this good?
> 
> image

This one looks good to me!

Thanks @mihirKachroo!

**Review by mica000 — APPROVED** [review]

File: src/app/components/balance.tsx (hunk 1 above)
Comment (mihirKachroo) [reply src/app/components/balance.tsx]: I use size "c2" from the Caption component in components/typography. It's font size 12 from what I remember.

**Comment by mihirKachroo** [comment]
New popup header with changes from dev:

image

**Comment by markmhendrickson** [comment]
@edu-stx to review and prepare for cloning and merging

**Comment by edu-stx** [comment]
Cloned to https://github.com/hirosystems/wallet/pull/3473 to run CI, and add final touches if any needed.

**Comment by edu-stx** [comment]
Thanks for your contribution @mihirKachroo. We've merged your commits in the branch above.
